releasenotes/notes/fix-race-condition-on-KubeUtil.rawConnectionInfo-254c4d68c325dd8d.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Protect KubeUtil.rawConnectionInfo map against race update.
```

Comment:
late to the party but this changelog is too specific, should more be user-facing like:
> Fix a crash that could occur when having trouble connecting to the Kubelet

Reply:
I will update it in a new PR.